iscript/tests/test_autograph.py

```diff
@@ -9,19 +9,28 @@
 from scriptworker_client.utils import makedirs
 
 from iscript.exceptions import IScriptError
+from iscript.mac import App
 import iscript.autograph as autograph
 
 # helper constants, fixtures, functions {{{1
 TEST_DATA_DIR = os.path.join(os.path.dirname(__file__), "data")
 
 
+@contextmanager
+def does_not_raise():
+    yield
+
+
 @pytest.fixture(scope="function")
 def key_config():
     return {
         "widevine_url": "https://autograph-hsm.dev.mozaws.net",
         "widevine_user": "widevine_user",
         "widevine_pass": "widevine_pass",
         "widevine_cert": "widevine_cert",
+        "langpack_url": "https://autograph-hsm.dev.mozaws.net/langpack",
+        "langpack_user": "langpack_user",
+        "langpack_pass": "langpack_pass",
```

Comment:
let's update `config_example.yaml` and https://github.com/mozilla-releng/scriptworker-scripts/wiki/Testing-iscript#script_configyaml